analyses/tp53_nf1_score/10-tp53-tumor-purity-threshold.Rmd

```diff
@@ -336,17 +336,19 @@ score_plot <- plot_tp53_status(tp53_thresholded_df, tp53_score, "TP53 Score")
 score_plot
 ```
 
-Here, the P-value is 0.39 which is still not significant. 
+Here, the P-value is 0.39 which is still not significant.
 Results are therefore consistent.
 
 We'll export the thresholded version of this plot, `score_plot`:
 
 ```{r}
 ggsave(
-  tp53_score_status_pdf, 
-  score_plot, 
+  tp53_score_status_pdf,
+  score_plot,
   width = 4,
-  height = 4
+  height = 4,
+  # needed for compilation
+  useDingbats = FALSE
```

Comment:
I'm putting this comment here, because I can't really comment on the line I want to comment on or the figure, but I find the varying alpha in these figures to be a bit disconcerting. It made me wonder if there was meaning to the alpha or if there were stacked points. Is there a reason we can't just set all to alpha = 0.25?

Reply:
The varying `alpha` in these figures is a strategy I used to make the points visible. Originally `alpha` was set to 0.25, but because the first two distributions have VERY low N (and hence no overlapping points), it was barely possible to actually see the points, just way too light. But, if you increase alpha, the third distribution starts to look wild with its much higher N and becomes too opaque. This was my middle ground..if you have other suggestions here I'll take em!

Reply:
Here are the panels. The "activated" points are just hard to see/find, but looking at these plots fresh today, maybe it's not as _too_ problematic as I was worried about since the violin and median +/ IQR guide are there to help.

[tp53-fpkm-status_tumor-purity-threshold.pdf](https://github.com/AlexsLemonade/OpenPBTA-analysis/files/10814825/tp53-fpkm-status_tumor-purity-threshold.pdf)
[tp53-score-status_tumor-purity-threshold.pdf](https://github.com/AlexsLemonade/OpenPBTA-analysis/files/10814826/tp53-score-status_tumor-purity-threshold.pdf)

Reply:
I don't find those bad at all. Most people will be looking at the summaries from the violin & IQR, as you note.